knox.go

```diff
@@ -52,15 +61,62 @@ func (c *knoxClient) RequestCredential(cred_type string) (credential_adapter.Ver
 	return cred, nil
 }
 
-func (c *knoxClient) PresentCredential(cred credential_adapter.VerifiableCredential) error {
+func (c *knoxClient) PresentCredential(creds ...model.SerializedDocument) error {
+
+	challenge, err := c.ca.CreatePresentationChallenge()
+	if err != nil {
+		return err
+	}
+	fmt.Printf("Challenge: %s, %s\n", challenge.Nonce, challenge.Url)
+	converted_creds := make([]map[string]interface{}, len(creds))
+
+	for i, cred := range creds {
+		var converted_cred map[string]interface{}
+		err = json.Unmarshal(cred, &converted_cred)
+
+		if err != nil {
+			return err
+		}
+
+		converted_creds[i] = converted_cred
+	}
+
+	vp := map[string]interface{}{
+		"@context":             []string{"https://www.w3.org/2018/credentials/v1", "https://www.w3.org/2018/credentials/examples/v1"},
+		"type":                 []string{"VerifiablePresentation"},
+		"verifiableCredential": converted_creds,
+	}
+
+	proc := ld.NewJsonLdProcessor()
```

Comment:
NewJSONLdProcessor according to a standard convention

Reply:
This is an external library so we have no control over what they name things

Reply:
@sKudryashov Good to resolve this?